sklearn/model_selection/_split.py

```diff
@@ -1050,7 +1050,7 @@ class StratifiedShuffleSplit(BaseShuffleSplit):
     >>> y = np.array([0, 0, 1, 1])
     >>> sss = StratifiedShuffleSplit(n_iter=3, test_size=0.5, random_state=0)
     >>> sss.get_n_splits(X, y)
-    3
+    3n
```

Comment:
> why the docstring didn't seem to be run.

Seems like nosetests ignores files starting with an underscore. Maybe we should use `ignore-files=DONTIGNOREANYFILES` as was mentioned [here](http://stackoverflow.com/questions/17863992/force-nosetests-to-find-doctests-in-modules-starting-with-underscore)